spec/formatting.md

```diff
@@ -235,6 +242,24 @@ rather than the _expression_ in the _selector_ or _pattern_.
 
 _Pattern selection_ is not supported for _fallback values_.
 
+### Expression Attribute Resolution
+
+The only _expression attribute_ that MUST be supported is `@locale`.
+If its value is set by a _literal_,
+it MUST be a comma-delimited sequence of IETF BCP 47 language tags.
```

Comment:
Do we wish to further constrain the language tags to be Unicode Locale Identifiers (ULIs)?

This text is non-specific about the conformance level of the language tags. Is well-formed enough? Valid? Canonicalized?

Reply:
I'm very happy to take input on this; I'm honestly not sure what the right specification here would be.